.wiki/DockerCompose.md

```diff
@@ -0,0 +1,23 @@
+# Quickstart with docker-compose
+
+Make sure you have already installed both [Docker Engine](https://docs.docker.com/get-docker/) and
+[Docker Compose](https://docs.docker.com/compose/install/).
+Then, obtain `docker-compose.yaml` file from the repository:
+you can [open it from the browser](https://github.com/artipie/artipie/blob/master/docker-compose.yaml),
+copy content and save it locally or use [git](https://git-scm.com/) and [clone](https://git-scm.com/docs/git-clone) the repository.
+As soon as Docker Compose is installed and `docker-compose.yaml` file is retrieved, open command line,
+`cd` to the location with the compose file and run Artipie service:
+
+```bash
+docker-compose up
+```
+
+It'll start a new Docker containers with latest Artipie and Artipie dashboard service images.
```

Comment:
"It'll start a new Docker containers" - "a" cannot be used with plural, so I suggest to remove this "a"